### DISCUSSION TITLE
The game crashed when using datapack to ban world generation

### DISCUSSION THREAD

**Discussion by CoderTCY** [FabricMC/fabric-api#3293] [FAQ] [answered]
### Description

I was trying to use the feature of filter of the datapack added in 1.19. When I banning world generation of the mod, It crashed. I find this in the log:
`java.lang.IllegalArgumentException: Couldn't find registry entry for ResourceKey[minecraft:worldgen/placed_feature / create:striated_ores_overworld]`
(I used Create mod)
I think it is caused by mod datapacks conbined to "Fabric Mods" datapack.
Also, how can I report it as a bug issue?

**Comment by CoderTCY** [comment]
The content of pack.mcmeta file:
```
{
    "pack": {
        "pack_format": 15,
        "description": "Disable everything in Create in survival mode"
    },
    "filter": {
        "block": [
            {"namespace": "create:*"}
        ]
    }
}
```
And the data folder is empty.

**Comment by PepperCode1** [comment, marked as answer]
It is not reasonable to expect Minecraft to not crash after removing all of a mod's data or Minecraft's data. This is not a bug with Fabric API, Create, or Minecraft; this behavior is entirely expected. If your goal is to disable Create's worldgen, it has a configuration option that does this.

**Reply by CoderTCY** [reply]
Well, I want to remove these in only one save...
And it won't crash when I was using Forge.
